### PR TITLE
Add published biomass burning info from ESGF

### LINKED ISSUE #128: Biomass burning data publication
Issue for tracking publication of the biomass burning data

### PR DESCRIPTION
**Pull request by znichollscr**
## Description

Workaround for issues defined in #133 
Fix #128 

## Checklist

Please confirm that this pull request has done the following:

- [x] Documentation added (where applicable)
- [x] Changelog item added to `changelog/`

### CODE REVIEW

File: docs/database-views/input4MIPs_delivery-summary_CMIP6Plus.html [hunk 1]
```diff
@@ -60,7 +60,7 @@ <h4>
       <td><a href='http://www.globalfiredata.org' target='_blank'>Open biomass burning emissions</a></td>
       <td>DRES-CMIP-BB4CMIP7-1-0</td>
       <td>Data in preparation and final metadata checks</td>
```
Comment (durack1) [inline docs/database-views/input4MIPs_delivery-summary_CMIP6Plus.html:62]: @znichollscr need to figure out where to catch this text and update it to "Preliminary dataset available"

**Review by durack1 — APPROVED** [review]
@znichollscr close, but I have run out of time - need to tweak ../docs/contributing.md to add these final flourishes/steps to get stuff sorted

**Comment by durack1** [comment]
@znichollscr over to you

**Comment by znichollscr** [comment]
@durack1 updated now. You can see the fixes I made in the commits. The human summary is:

- there were no new entries in the database, so it looks like the workaround script hadn't been run or the results hadn't been committed
- I've updated the docs to hopefully clarify the process, feel free to add any tweaks in a follow up MR (I know you're keen to get this merged and live, so I'm just going to hit the button)

cc @eleanororourke @vnaik60, update should be live now (or in 2 minutes)

**Comment by durack1** [comment]
@znichollscr probably worth a patch bump, no?

**Comment by znichollscr** [comment]
Can't hurt, go for it

**Pull request merged**